examples/scripts/test-computeCDF.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This script shows how to use the computeCDF method in order
+to compute exact probabilities using Distribution arithmetic.
+"""
+
+# Creates a reliability problem RP8.
+import otbenchmark as otb
+import numpy as np
+
+
+def ComputeReferenceProbability(problem, verbose=False):
+    event = problem.getEvent()
+    antecedent = event.getAntecedent()
+    distribution = antecedent.getDistribution()
+    # Check copula
+    copula = distribution.getCopula()
+    if not copula.hasIndependentCopula():
+        raise Exception("Copula is not independent.")
+    # Get marginal distribution
+    dimension = distribution.getDimension()
+    for i in range(dimension):
+        marginal = distribution.getMarginal(i)
+        marginal_statement = "X%d=marginal" % (i + 1)
+        if verbose and i < 5:
+            print(marginal)
+        exec(marginal_statement)
+    # Prepare constants and functions
+    PI_ = np.pi
+    _ = PI_  # To fool flake8
+
+    def SQRT(x):
+        return np.sqrt(x)
+
+    def MIN(x, y):
+        return min(x, y)
+
+    def ABS(x):
+        return abs(x)
+
+    # Get the formula
+    function = event.getFunction()
```

Comment:
If all the functions in your events are SymbolicFunction, you could store the formula in the definition of the problem instead of trying to recover it from the string serialization.